src/utils/rootfs.py

```diff
@@ -1,17 +1,18 @@
-'''
-Copyright (c) 2017-2018 VMware, Inc. All Rights Reserved.
-SPDX-License-Identifier: BSD-2-Clause
-'''
+#
+# Copyright (c) 2017-2018 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+#
+"""
+Operations to mount container filesystems and run commands against them
+"""
+
 import hashlib
 import logging
 import os
 import subprocess  # nosec
 import tarfile
 
 from . import constants
```

Comment:
When I ran prospector on this, I got this message: `pylint: relative-beyond-top-level / Attempted relative import beyond top-level package`

The file has since been moved to the `tern` folder and is now under a module name. So this can be solved using:
Using `from tern.utils import constants`